test/sync_replay.json

```diff
@@ -59,18 +59,18 @@
     "data": "{\"transactionId\":3,\"key\":\"/todo/14323534\"}",
     "responseBody": "{\"has\":true,\"value\":{\"complete\":false,\"id\":14323534,\"listId\":1,\"order\":10000,\"text\":\"Test\"}}"
   },
-  {
-    "dbName": "sync",
```

Comment:
Note to self: Maybe add to the log when the request is sent instead of when the request is received?